source/dbo/Views/Power BI Reporting/vwMissingHospitalMappings.sql

```diff
@@ -0,0 +1,61 @@
+﻿CREATE VIEW [dbo].[vwMissingHospitalMappings]
+AS
+
+-- LTBR hospital is not in migration mapping
+SELECT CONCAT(dp_PatientID, '-', dp_DiseasePeriod) AS OldNotificationId,
+	'LTBR' AS [Source],
+	dp_NotifiedDate AS NotificationDate,
+	CAST(dp_NotifyingClinicID AS NVARCHAR(50)) AS HospitalId,
+	lh_HospitalName COLLATE Latin1_General_CI_AS AS HospitalName,
+	residencePhec.[Name] AS ResidenceRegion,
+	CAST(NULL AS NVARCHAR) AS TreatmentRegion
+FROM [$(migration)].dbo.LTBR_DiseasePeriod dp
+	INNER JOIN [$(migration)].dbo.LtbrAddresses a ON a.OldNotificationId = CONCAT(dp_PatientID, '-', dp_DiseasePeriod)
+	LEFT JOIN [$(LTBR)].dbo.sbt_LondonHospitals lh ON dp_NotifyingClinicID = lh_HospitalID
+	LEFT JOIN [$(NTBS_R1_Geography_Staging)].dbo.Reduced_Postcode_file rpcd ON rpcd.Pcode = a.Postcode
+	LEFT JOIN [$(NTBS_R1_Geography_Staging)].dbo.LA_to_PHEC residenceLA ON residenceLA.LA_Code = rpcd.LA_Code
+	LEFT JOIN [$(NTBS)].ReferenceData.PHEC residencePhec ON residenceLA.PHEC_Code = residencePhec.Code
+WHERE (dp_NotifyingClinicID IS NULL OR dp_NotifyingClinicID NOT IN (SELECT LtbrId FROM [$(migration)].dbo.LtbrHospitalsMapping))
+	AND YEAR(dp_NotifiedDate) IN (SELECT NotificationYear FROM vwNotificationYear)
+
+UNION
+
+-- ETS/NTBS Hospital mapped to from LTBR hospital does not exist in NTBS
+SELECT CONCAT(dp_PatientID, '-', dp_DiseasePeriod) AS OldNotificationId,
+	'LTBR' AS [Source],
+	dp_NotifiedDate AS NotificationDate,
+	CAST(dp_NotifyingClinicID AS NVARCHAR(50)) AS HospitalId,
+	lhm.LtbrName AS HospitalName,
+	residencePhec.[Name] AS ResidenceRegion,
+	CAST(NULL AS NVARCHAR) AS TreatmentRegion
+FROM [$(migration)].dbo.LTBR_DiseasePeriod dp
+	INNER JOIN [$(migration)].dbo.LtbrAddresses a ON a.OldNotificationId = CONCAT(dp_PatientID, '-', dp_DiseasePeriod)
+	LEFT JOIN [$(migration)].dbo.LtbrHospitalsMapping lhm ON dp_NotifyingClinicID = lhm.LtbrId
```

Comment:
Could we make this an inner join, since the cases where the `dp_NotifyingClinicID` is not found in this table are already included in the select above?